pyproject.toml

```diff
@@ -1,7 +1,14 @@
 [build-system]
-requires = ["setuptools>=39.2.0", "wheel"]
+requires = ["setuptools", "wheel"]
```

Comment:
I'm pretty sure we want to continue specifying a version here since we use `attr:` for getting the version in `setup.cfg`.